pasta_eln/GUI/projectTreeView.py

```diff
@@ -171,6 +172,61 @@ def tree2Clicked(self) -> None:
     return
 
 
+  def dragEnterEvent(self, event:QMouseEvent) -> None:
+    """
+    Overwrite default: what happens if you drag an item
```

Comment:
Did you mean "override"? Is this a base class function which is been overridden here in this clasS?

Reply:
Yes